apis/cloudantv1/v1alpha1/cloudantdatabase_types.go

```diff
@@ -25,26 +25,27 @@ import (
 // CloudantDatabaseParameters are the configurable fields of a CloudantDatabase.
 type CloudantDatabaseParameters struct {
 
+	// should this json tag be 'json:"db"' ??
```

Comment:
right, to be consistent with the API. It should be immutable as well, but not omitempty as it should be always filled in